deployments/infrastructure.nix

```diff
@@ -41,15 +43,6 @@ in {
       };
       groups.staging = {};
 
-      users.production = {
-        description     = "cardano production";
-        group           = "production";
-        createHome      = true;
-        isNormalUser = true;
-        openssh.authorizedKeys.keys = [];  # this account is obsolete
-      };
-      groups.production = {};
-
```

Comment:
№4

Do we want to rename `live-production` back to `production`?

Reply:
Nah, let's leave it as is. Not worth the time.